src/components/EventTile/EventTile.jsx

```diff
@@ -0,0 +1,47 @@
+import './EventTile.scss';
+
+import React, { PureComponent, PropTypes } from 'react';
+
+import cxHelpers from 'lib/decorators/classNameHelpers';
+import Icon from 'components/Icon';
+
+@cxHelpers("EventTile")
+class EventTile extends PureComponent {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    members: PropTypes.array.isRequired,
+    image_url: PropTypes.string
+  };
+
+  renderMembers(members) {
+    return members.map(member => {
+      console.log({member})
+      return <span>O</span>;
+    })
+  }
+
+  render(){
+    const { name, image_url, members, style } = this.props;
+    const backgroundStyle = image_url
```

Comment:
Another multi line ternary, and for something like this you could argue it should be its own method.  Something like "setBackground"